app/llama_sensei/backend/qa/generate_answer.py

```diff
@@ -1,31 +1,46 @@
+import time
 from datetime import datetime
 
+import numpy as np
 from datasets import Dataset
+from langchain_community.embeddings import SentenceTransformerEmbeddings
+from langchain_community.utilities import DuckDuckGoSearchAPIWrapper
 from langchain_groq import ChatGroq
+from llama_sensei.backend.add_courses.embedding.get_embedding import Embedder
```

Comment:
don't import like this, 2 backend will later be separated
use self.embedder instead (see comment below)